MapboxCoreNavigation/VisualInstructionFormatter.swift

```diff
@@ -27,4 +27,56 @@ public class VisualInstructionFormatter: NSObject {
             return nil
         }
     }
+    
+    /**
+     Creates the optimal text to be displayed given a `RouteLeg` and `RouteStep` divided into primary and secondary string.
+     */
+    public func strings(leg: RouteLeg?, step: RouteStep?) -> (String?, String?) {
```

Comment:
Turn this return type into a typealias to the tuple, so that we can document it better.

Reply:
By the way, a tuple return type won’t bridge to Objective-C, although I don’t know whether that’s a concern at the moment. The Objective-C way of doing things would be to have two separate methods, one for the primary text and the other for the secondary text. That would be far easier to implement after https://github.com/mapbox/mapbox-navigation-ios/pull/745#discussion_r148137222.

Reply:
This method is unused so I just removed it.